tailwind.config.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
         'zinc-dark-purple': '#1F1735',
         'zinc-hot-pink': '#F43EF3',
         'zinc-pastel-pink': '#FE9BFF',
+        'zinc-teal-blue': '#38ADB5',
```

Comment:
this change makes `zinc-hot-pink` not used anywhere in the site. But I leave the color defined here so we CAN use it for emphasis in the future for more redesigns.